src/animators/baseAnimator.ts

```diff
@@ -63,7 +63,7 @@ export module Animators {
       return time;
     }
 
-    public animate(selection: any, attrToProjector: AttributeToProjector): D3.Transition.Transition {
+    public animate(selection: d3.Selection<any>, attrToAppliedProjector: AttributeToAppliedProjector) {
```

Comment:
an `AppliedProjector` has the `Dataset` rolled into it, so the signature is different:
```Typescript
/**
 * Retrieves scaled datum property.
 */
export type _Projector = (datum: any, index: number, dataset: Dataset) => any;

export type AppliedProjector = (datum: any, index: number) => any;
```